en-US/snippets/2017/jul2017.lang

```diff
@@ -0,0 +1,145 @@
+<a>Mozilla is a global non-profit</a> dedicated to putting you in control of your online experience and shaping the future of the web for the public good. #nowyouknow
+
+
+# Desktop Snippet
+# Link: https://www.facebook.com/sharer/sharer.php?u=https://www.mozilla.org/firefox/new/
+# Squad is a word used to refer to a close group of friends.
+# Feel free to swap fries for popular simple local food.
+;Good friends share fries. Great friends share fries and the last independent browser. <a>Share Firefox with your squad on Facebook.</a>
+Good friends share fries. Great friends share fries and the last independent browser. <a>Share Firefox with your squad on Facebook.</a>
+
+
+# Desktop Snippet
+# Link: https://support.mozilla.org/products/firefox/protect-your-privacy
+;Cookies and phishing might sound fun but when it comes to your privacy online, it pays to <a>take them seriously.</a>
+Cookies and phishing might sound fun but when it comes to your privacy online, it pays to <a>take them seriously.</a>
+
+
+# Desktop Snippet
+# Link: https://support.mozilla.org/products/firefox/protect-your-privacy
+# Cinch means very easy.
+;Protecting yourself online can be a lot of work. Luckily, <a>Firefox gives you the tools that make privacy a cinch.</a>
+Protecting yourself online can be a lot of work. Luckily, <a>Firefox gives you the tools that make privacy a cinch.</a>
+
+
+# Desktop Snippet
+# Link: https://support.mozilla.org/products/firefox/protect-your-privacy
+;Always feel like somebody is watching you? Control your data, block malware and outfox the trackers with <a>Firefox’s privacy protection tips.</a>
+Always feel like somebody is watching you? Control your data, block malware and outfox the trackers with <a>Firefox’s privacy protection tips.</a>
+
+
+# Desktop Snippet
+# https://support.mozilla.org/kb/get-started-firefox-overview-main-features
+;<a>Become a power Firefox user.</a> Discover customization tricks and something we call, THE AWESOME BAR.
+<a>Become a power Firefox user.</a> Discover customization tricks and something we call, THE AWESOME BAR.
+
+
+# Desktop Snippet
+# https://support.mozilla.org/kb/get-started-firefox-overview-main-features
+;Whether you’re new to Firefox or just need a refresher, here’s a quick <a>guide to all the awesomeness Firefox comes with</a> right out of the box.
+Whether you’re new to Firefox or just need a refresher, here’s a quick <a>guide to all the awesomeness Firefox comes with</a> right out of the box.
+
+
+# Desktop Snippet
+;<a>Fast. Small. Private.</a>
+<a>Fast. Small. Private.</a>
+
+
+# Desktop Snippet
+You don't need to sacrifice a lot of data to experience the best in private browsing. Send Firefox Focus to your phone.
+;You don't need to sacrifice a lot of data to experience the best in private browsing. Send Firefox Focus to your phone.
```

Comment:
Indeed, the error page doesn't say anything because it only checks the reference strings, and ignore translations. Still worth fixing it.